client/src/components/Styling/Theme.js

```diff
@@ -0,0 +1,97 @@
+const hivestack =     
+{
+  global: {
+    colors: {
+      brand: "#F0BE27",
+      border: {
+        dark: "#ebebeb",
+        light: "#444444"
+      },
+      background: "#F4F4F2",
+      placeholder: "rgba(68, 68, 68, 0.5)",
+      control: {
+        dark: "#F0BE27",
+        light: "#0F41D8"
+      },
+      accent_1: "#ffdd00",
+      accent_2: "#b39b00",
+      accent_3: "#fff7bf",
+      accent_4: "#ffee80",
+      neutral_1: "#3c0099",
+      neutral_2: "#2a006b",
+      neutral_3: "#d8bfff",
+      neutral_4: "#b280ff",
+      neutral_5: "#ba0098"
+    },
+    elevation: {
+      none: "none",
+      xsmall: "0px 1px 2px rgba(68, 68, 68, 0.5)",
+      small: "0px 2px 4px rgba(68, 68, 68, 0.5)",
+      medium: "0px 3px 8px rgba(68, 68, 68, 0.5)",
+      large: "0px 6px 12px rgba(68, 68, 68, 0.5)",
+      xlarge: "0px 8px 16px rgba(68, 68, 68, 0.5)"
+    },
+    drop: {
+      background: "rgb(233, 233, 229)",
+      shadow: "0px 3px 8px rgba(68, 68, 68, 0.5)",
+      border: {
+        radius: "3px"
+      }
+    },
+    input: {
+      border: {
+        radius: "16px"
+      }
+    },
+    font: {
+      family: "'Roboto', sans-serif",
```

Comment:
why is this here again when you already have it in the other component?

Reply:
I was trying to find a better way to implement the custom theme, it is still a work in progress

Reply:
The theme styling was created to reduce duplicate code blocks with the custom theme, still working on implementing that. Face has also been removed.